README.md

```diff
@@ -55,6 +55,11 @@ const kilometersRounded = miles.as('kilometers', 2); // 296119.3
 | us gallon | `usgal` `usgallon` `usgallons` |
 | imperial gallon | `gal` `gallon` `gallons` |
 
+| Fuel Consumption | aliases |
```

Comment:
this should be `Fuel Economy` interestingly enough consumption is the inverse of economy, who knew :man_shrugging: 


https://energyeducation.ca/encyclopedia/Fuel_consumption_vs_fuel_economy

Reply:
jfsdifhsadkfj i saw this, forgot when I was typing this though. my bad!!!!